coreless-ui/frontend/src/views/forms/customer-offer/CustomerOffer.js

```diff
@@ -91,9 +91,11 @@ export default class CustomerOffer extends React.Component {
     const requestOptions = {
       method: 'POST',
       headers: { 'Content-Type': 'application/json' },
-      body: JSON.stringify({ procedure: { customerReference: this.state.customerFirstName } }),
+      body: JSON.stringify({
+        customerOfferProcedureInstanceRecord: { customerReference: this.state.customerFirstName },
```

Comment:
please format it better
```suggestion
        customerOfferProcedureInstanceRecord: { 
             customerReference: this.state.customerFirstName 
        },
```

Reply:
fixed, thank you